torch/csrc/jit/passes/tensorexpr_fuser.h

```diff
@@ -60,6 +60,7 @@ TORCH_API bool usedOnlyInSize(Value* v);
 TORCH_API Value* broadcastSizes(at::ArrayRef<Value*> sizes, AliasDb* db);
 
 namespace tensorexpr {
+TORCH_API const OperatorSet& supported_eltwise_set();
 TORCH_API bool isSupported(Node* node);
```

Comment:
While we're here let's fix the naming conventions so that we consistently use lowerCamelCase.